backend/wmg/pipeline/integrated_corpus/job.py

```diff
@@ -48,7 +48,11 @@ def build_integrated_corpus(dataset_directory: List, corpus_path: str):
     """
     with tiledb.scope_ctx(create_ctx()):
         dataset_count = len(os.listdir(dataset_directory))
-        dataset_gene_mapping = {}
+        if os.path.exists(f"{corpus_path}/{DATASET_TO_GENE_IDS_NAME}.json"):
```

Comment:
If we resume integrated corpus build, we should start from the extant json.

Reply:
There's no case where this file exists and we're trying to **regenerate** integrated corpus and not **resume**?

Reply:
There's no such case, currently. In that case, you would need to delete the current integrated corpus folder and start again. The above update was to match current behavior.